### DISCUSSION TITLE
[auth] useEmulator

### DISCUSSION THREAD

**Discussion by radvansky-tomas** [firebase/flutterfire#3969] [Feature request] [answered]
Can you please expose useEmulator() method to allow auth emulator for all platforms?

https://firebase.google.com/docs/emulator-suite/connect_auth

```
firebase.auth().useEmulator('http://localhost:9099/');
Auth.auth().useEmulator(withHost:"localhost", port:9099)
FirebaseAuth.getInstance().useEmulator('10.0.2.2', 9099);
```

**Comment by nipunasudha** [comment]
+1

**Comment by newit-huytt** [comment]
+1

**Comment by Salakar** [comment, marked as answer]
PR is ready for testing, see the description if you'd like to try it out and give feedback: https://github.com/FirebaseExtended/flutterfire/pull/4263

**Comment by deep-randhawa** [comment]
Any updates on this?
